websockets/Streaming.WebSocket.Samples/WebSocketSample.cs

```diff
@@ -196,13 +196,20 @@ private async Task ReauthorizeWhenNeeded(DateTime tokenExpiryTime, CancellationT
             }
         }
 
+        private HttpClient CreateHttpClient()
+        {
+            var handler = new HttpClientHandler { AllowAutoRedirect = false, AutomaticDecompression = System.Net.DecompressionMethods.GZip };
+            return new HttpClient(handler);
```

Comment:
This can be added here as well: httpClient.DefaultRequestHeaders.ExpectContinue = false;
However, isn't the default not "false", making this statement redundant?

Reply:
moved httpClient.DefaultRequestHeaders.ExpectContinue = false; here.

The default value of httpClient.DefaultRequestHeaders.ExpectContinue is null. As long as it is null, the HttpClient will include the expect 100-continue.  So I set it to false here.